src/auth-backend.js

```diff
@@ -23,8 +29,90 @@ export default class AuthBackend extends Backend {
 		}
 	}
 
-	async login () {}
-	async logout () {}
+	/**
+	 * Log a user in, either passively (without triggering any login UI) or actively (with login UI)
+	 * @param [options] {object}
+	 * @param [options.passive] {boolean} - Do not trigger any login UI, just return the current user if already logged in
+	 */
+	async login ({passive = false, ...rest} = {}) {
+		if (this.ready) {
+			await this.ready;
+		}
+
+		if (this.isAuthenticated()) {
+			return this.getUser();
+		}
+
+		await passiveLogin(rest);
+
+		if (this.isAuthenticated()) {
+			try {
+				await this.getUser()
```

Comment:
Shouldn't we dispatch the `mv-login` event here too? And update the permissions?

Reply:
this does not return, it continues on, and fires `mv-login` in line 66 below